script.js

```diff
@@ -0,0 +1,31 @@
+const menuButton = document.querySelector('.menu-button');
+const closeButton = document.querySelector('.close-button');
+const mobileMenu = document.querySelector('.mobile-menu');
+const mobileMenuItem = document.querySelectorAll('.mobile-menu-item');
+
+function toggleMobileMenu() {
+  mobileMenu.style.display = 'flex';
+}
+
+function closeMobileMenu() {
+  mobileMenu.style.display = 'none';
+}
+
+function handleMobileMenuItemClick(event) {
+  const { target } = event;
+  const sectionId = target.getAttribute('href').substring(1);
+
+  toggleMobileMenu();
+
+  document.querySelector(sectionId).scrollIntoView({
+    behavior: 'smooth',
+    block: 'start',
+  });
+}
```

Comment:
Well done on implementing the navigation making the menu navigate to the right section when an item is clicked 💯 but I noticed that the menu is still open after navigating, please ensure that the menu closes after that to provide a better user experience. 📱 

You can do that by adding an event listener to each one of these and making sure that the menu closes when they're clicked. Read more about event listeners [here](https://www.w3schools.com/js/js_htmldom_eventlistener.asp). 🔗 

In your case here you can just call the function to close the menu ❌